docs/tuf-spec.md

```diff
@@ -491,9 +496,9 @@ Version: **1.0 (Draft)**
    used to sign documents.  The type determines the interpretation of
    KEYVAL.
```

Comment:
How does the type determine the interpretation of KEYVAL? Type here implies ecdsa/rsa/ed25519, not public/private. Is public/private meant?